cpp/tests/copying/slice_tests.cpp

```diff
@@ -476,11 +476,30 @@ TEST_F(SliceTableCornerCases, MiscOffset)
   cudf::test::fixed_width_column_wrapper<int32_t> col2{
     {3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3,
      3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3},
-    {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
-     1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0}};
+    {true, true, true, true, true, true, true, true, true, true,  true,  true, true,
+     true, true, true, true, true, true, true, true, true, true,  true,  true, true,
+     true, true, true, true, true, true, true, true, true, false, false, false}};
   cudf::test::fixed_width_column_wrapper<int32_t> col3{
     {3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3},
-    {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0}};
+    {true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     true,
+     false,
+     false,
+     false}};
```

Comment:
nit: same comment about validity mask. readability is better, if the each data aligns with its validity too.